spire-flex/templates/agent-configmap.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $.Release.Name }}-agent-config
+  namespace: {{ $.Release.Namespace }}
```

Comment:
Missing "spire-flex.labels" stuff.

namespace is optional as that is what you will get by default out of helm. though doesn't hurt anything. just an fyi.

Reply:
I need the namespace to be rendered so I can unit test it.

Reply:
Let's handle labels when we have multiple objects.  I want a consistent labeling scheme across all of the objects, with support for default Helm labels and custom labels.